pkg/controller/reference/reconciler.go

```diff
@@ -0,0 +1,276 @@
+// Copyright 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
GitHub shows this change as new file instead of moved.
```suggestion
// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```